Assets/Scripts/EOSLobbyManager.cs

```diff
@@ -1529,65 +1535,137 @@ public void PressToTalk(KeyCode PTTKeyCode, OnLobbyCallback TogglePressToTalkCom
+                rtcAudioHandle.UpdateReceiving(ref recevingOptions, MuteMemberCompleted, OnRTCRoomUpdateReceivingCompleted);
+            }
+        }
 
-                    UpdateReceivingOptions recevingOptions = new UpdateReceivingOptions()
-                    {
-                        LocalUserId = EOSManager.Instance.GetProductUserId(),
-                        RoomName = CurrentLobby.RTCRoomName,
-                        ParticipantId = targetUserId,
-                        AudioEnabled = lobbyMember.RTCState.IsLocalMuted
-                    };
+        /// <summary>
+        /// Set the Mute Status of the Local Member.
+        /// The local user sets their mute status to <paramref name="micOn"/>,
+        /// either becoming muted or unmuting themself.
+        /// While muted, other users in the lobby will not be able to hear the local user.
+        /// </summary>
+        /// <param name="shouldAudioBeEnabled">
+        /// If true, the local user unmutes themself.
+        /// If false, the local user mutes themself, and no longer sends audio.
+        /// </param>
+        /// <param name="MuteLocalMemberCompleted">Callback when MuteLocalMember is completed</param>
+        public void MuteLocalMember(bool shouldAudioBeEnabled, OnLobbyCallback MuteLocalMemberCompleted)
+        {
+            UpdateSendingOptions sendOptions = new UpdateSendingOptions()
+            {
+                LocalUserId = EOSManager.Instance.GetProductUserId(),
+                RoomName = GetCurrentLobby().RTCRoomName,
+                AudioStatus = shouldAudioBeEnabled ? RTCAudioStatus.Enabled : RTCAudioStatus.Disabled
+            };
 
-                    Debug.LogFormat("Lobbies (ToggleMute): {0} remote player {1}", recevingOptions.AudioEnabled ? "Unmuting" : "Muting", targetUserId);
+            Debug.LogFormat("Lobbies (MuteLocalMember): Setting self audio output status to {0}", sendOptions.AudioStatus == RTCAudioStatus.Enabled ? "Unmuted" : "Muted");
 
-                    rtcAudioHandle.UpdateReceiving(ref recevingOptions, ToggleMuteCompleted, OnRTCRoomUpdateReceivingCompleted);
-                }
-            }
+            EOSManager.Instance.GetEOSRTCInterface().GetAudioInterface().UpdateSending(ref sendOptions, MuteLocalMemberCompleted, OnRTCRoomUpdateSendingCompleted);
+        }
+
+        /// <summary>
+        /// Sets the Deafen Status of the Local Member.
+        /// If <paramref name="shouldBeDefeaned"/> is true, then the user won't be able to hear other users.
+        /// </summary>
+        /// <param name="shouldBeDefeaned">
+        /// If true, the local user should become defeaned.
+        /// Otherwise, undeafen the user.
+        /// </param>
+        /// <param name="DeafenCompleted">Callback when DeafenLocalMember is completed</param>
+        public void DeafenLocalMember(bool shouldBeDefeaned, OnLobbyCallback DeafenCompleted)
```

Comment:
Probably change the function name similarly to the `MuteMember` function, so that it indicates that the deafen state can be either set or un-set.

Reply:
Did some renaming.